documents/knowledge/elements.yaml

```diff
@@ -2452,7 +2452,7 @@
 - element: "⇩"
   name: Grade Down
   arity: 1
-  description: Indices of elements to sort in descending order / lowercase / decrement number twice
+  description: 'Indices of elements to sort in descending order / lowercase / decrement number twice'
```

Comment:
Why the quotes? Was there an issue with the YAML?

Reply:
No clue.